cmd/main.go

```diff
@@ -108,9 +115,11 @@ func main() {
 
 	printControllerMetaInfo()
 
+	ctx := ctrl.SetupSignalHandler()
+
 	var watchNamespaces = "WATCH_NAMESPACES"
```

Comment:
does this do anything in a multi-cluster setup. I presume not given it is an operator sdk thing

Reply:
Good question, I'd say no, but i haven't really thought about what this would actually mean if you are reconciling remote records. Would seem odd to reconcile all namespaces on a remote but limit to a single one on the local cluster. :thinking: 

The multi cluster manager does use the same options, i haven't dug through the code, but might be worth trying just to know for sure what it actually does if you set this.

Reply:
Need to think about this, currently this will impact on the behaviour. If `WATCH_NAMESPACES` is set, it is set on the local manager options which is used for any local watches, including the cluster secrets. This means that whatever  `CLUSTER_SECRET_NAMESPACE` is set to must also be in this list in order for it to discover local cluster kubeconfig secrets as expected.